addons/service/system/proftpd/package.mk

```diff
@@ -31,41 +31,39 @@ PKG_PRIORITY="optional"
 PKG_SECTION="service/system"
 PKG_SHORTDESC="Highly configurable GPL-licensed FTP server software"
 PKG_LONGDESC="Secure and configurable FTP server with SSL/TLS support"
-
 PKG_IS_ADDON="yes"
 PKG_ADDON_TYPE="xbmc.service"
 PKG_ADDON_PROVIDES=""
 PKG_AUTORECONF="no"
 PKG_ADDON_REPOVERSION="6.0"
-
 PKG_MAINTAINER="vpeter4 (peter.vicman@gmail.com)"
 
-ADDON_DIR="/storage/.kodi/addons/service.system.proftpd"
+pre_configure_target() {
+  export CFLAGS="$CFLAGS -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -I$SYSROOT_PREFIX/usr/include/ncurses -I$ROOT/$PKG_BUILD/.$TARGET_NAME/include/"
+  export LDFLAGS="$LDFLAGS -L$ROOT/$PKG_BUILD/.$TARGET_NAME/lib"
+
+  ADDON_DIR="/storage/.kodi/addons/service.system.proftpd"
 
-PKG_CONFIGURE_OPTS_TARGET="--enable-static \
-            --enable-openssl \
-            --with-modules=mod_tls \
-            --enable-nls \
-            --localedir=$ADDON_DIR/locale \
-            --enable-sendfile \
-            --enable-facl \
-            --enable-autoshadow \
-            --enable-ctrls \
-            --enable-ipv6 \
-            --enable-nls \
-            --enable-pcre \
-            --enable-largefile"
+  PKG_CONFIGURE_OPTS_TARGET="--enable-static \
```

Comment:
Changed back as it was before.
But can it be in configure_target() where is actually used?